packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -148,12 +158,31 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
             tabIndex={isFunction(onClick) ? 0 : -1}
             onClick={onClick}
             onKeyDown={onEnterPressed(onClick)}
-            {...restProps}
+            {...props}
         >
             {aspectRatio === AspectRatio.original ? (
                 <img className="lumx-thumbnail__image" src={image} alt={alt} loading={loading} />
             ) : (
-                <div className="lumx-thumbnail__background" style={style} />
+                <div className="lumx-thumbnail__background">
+                    <img
+                        ref={(f): FocusedImage | undefined => {
+                            return f
+                                ? new FocusedImage(f as ILumHTMLImageElement, {
+                                      debounceTime: 17,
+                                      focus,
+                                      updateOnWindowResize: true,
+                                  })
+                                : undefined;
+                        }}
```

Comment:
Could it be possible to have a static method `FocusImage.attach` that could be used like this:
```tsx
<img ref={FocusedImage.attach} data-focus-x={focus.x} data-focus-y={focus.y} />
```
Could the focus point be controlled with the `data` attributes?
Also I think it would be clearer to rename `focus` to `focusPoint` everywhere to not confuse the HTML focus with this image focus